features/prompt-registry.mdx

```diff
@@ -83,13 +80,22 @@ You can also optionally pass `version` to get an older version of a prompt. By d
 <CodeGroup>
 
 ```python Python SDK
-template_dict = promptlayer.prompts.get("my_template", version=3)
+template_dict = promptlayer.templates.get(
+  "my_template",
+  {
+    "version": 3
+  }
+)
 ```
 
 ```js JavaScript
-const template_dict = await promptlayer.prompts.get({
-  prompt_name: "my_template", version: 3
-});
+const template_dict = await promptlayer.templates.get(
```

Comment:
one line

Reply:
One line? Do you wane me to move print statement to one line?